src/tiled/scriptmodule.cpp

```diff
@@ -282,6 +306,63 @@ QJSValue ScriptModule::registerTool(const QString &shortName, QJSValue toolObjec
     return toolObject;
 }
 
+Tiled::ScriptMapFormatWrapper *ScriptModule::getMapFormat(const QString &shortName) const
+{
+    auto formats = PluginManager::objects<MapFormat>();
```

Comment:
```suggestion
    const auto formats = PluginManager::objects<MapFormat>();
```
(3 more times below)